### ISSUE TITLE
JTReg Test Failure : java/lang/invoke/condy/CondyTypeValidationTest.java

### ISSUE THREAD

**Issue by ben-walsh** [eclipse/openj9#4964]
Consistent failures ...

```
[2019-03-04T10:41:52.378Z] test CondyTypeValidationTest.testInvalidTypes("L/java/lang/Object"): failure
[2019-03-04T10:41:52.378Z] java.lang.IllegalArgumentException: malformed method descriptor: ()L/java/lang/Object
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.parseIntoClass(MethodType.java:432)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.parseIntoClasses(MethodType.java:496)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.fromMethodDescriptorString(MethodType.java:316)
[2019-03-04T10:41:52.378Z] 	at test.java.lang.invoke.lib.InstructionHelper.ldc(InstructionHelper.java:141)
[2019-03-04T10:41:52.378Z] 	at test.java.lang.invoke.lib.InstructionHelper.ldcDynamicConstant(InstructionHelper.java:114)
[2019-03-04T10:41:52.378Z] 	at test.java.lang.invoke.lib.InstructionHelper.ldcDynamicConstant(InstructionHelper.java:107)
[2019-03-04T10:41:52.378Z] 	at CondyTypeValidationTest.testInvalidTypes(CondyTypeValidationTest.java:66)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestRunner.privateRun(TestRunner.java:773)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestRunner.run(TestRunner.java:623)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestNG.run(TestNG.java:1018)
[2019-03-04T10:41:52.378Z] 	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:94)
[2019-03-04T10:41:52.378Z] 	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:54)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[2019-03-04T10:41:52.378Z] 	at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.Thread.run(Thread.java:825)
[2019-03-04T10:41:52.378Z] test CondyTypeValidationTest.testInvalidTypes("[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[I"): failure
[2019-03-04T10:41:52.378Z] java.lang.TypeNotPresentException: Type [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[I not present
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.nonPrimitiveClassFromString(MethodType.java:408)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.parseIntoClass(MethodType.java:437)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.parseIntoClasses(MethodType.java:496)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.fromMethodDescriptorString(MethodType.java:316)
[2019-03-04T10:41:52.378Z] 	at test.java.lang.invoke.lib.InstructionHelper.ldc(InstructionHelper.java:141)
[2019-03-04T10:41:52.378Z] 	at test.java.lang.invoke.lib.InstructionHelper.ldcDynamicConstant(InstructionHelper.java:114)
[2019-03-04T10:41:52.378Z] 	at test.java.lang.invoke.lib.InstructionHelper.ldcDynamicConstant(InstructionHelper.java:107)
[2019-03-04T10:41:52.378Z] 	at CondyTypeValidationTest.testInvalidTypes(CondyTypeValidationTest.java:66)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
[2019-03-04T10:41:52.378Z] 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestRunner.privateRun(TestRunner.java:773)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestRunner.run(TestRunner.java:623)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
[2019-03-04T10:41:52.378Z] 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
[2019-03-04T10:41:52.378Z] 	at org.testng.TestNG.run(TestNG.java:1018)
[2019-03-04T10:41:52.378Z] 	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:94)
[2019-03-04T10:41:52.378Z] 	at com.sun.javatest.regtest.agent.TestNGRunner.main(TestNGRunner.java:54)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2019-03-04T10:41:52.378Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[2019-03-04T10:41:52.378Z] 	at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.Thread.run(Thread.java:825)
[2019-03-04T10:41:52.378Z] Caused by: java.lang.ClassNotFoundException: [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[I
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.Class.forNameImpl(Native Method)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.Class.forName(Class.java:415)
[2019-03-04T10:41:52.378Z] 	at java.base/java.lang.invoke.MethodType.nonPrimitiveClassFromString(MethodType.java:406)
[2019-03-04T10:41:52.378Z] 	... 36 more
```

... observed with test https://github.com/ibmruntimes/openj9-openjdk-jdk12/blob/openj9/test/jdk/java/lang/invoke/condy/CondyTypeValidationTest.java against an OpenJDK12+OpenJ9 build (https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk12u/job/jdk12u-linux-x64-openj9/16/artifact/workspace/target/OpenJDK12U-jdk_x64_linux_openj9_2019-02-28-09-45.tar.gz).

See [consoleText.txt](https://github.com/eclipse/openj9/files/2926840/consoleText.txt).

**Comment by theresa-m** [comment]
looking at this as well

**Comment by fengxue-IS** [comment]
This is similar to failure mentioned in https://github.com/eclipse/openj9/issues/4966#issuecomment-469323276, Should check if #4919 fixes this.

**Comment by JasonFengJ9** [comment]
Verified manually that https://github.com/eclipse/openj9/pull/4919 makes this test pass.

**Comment by JasonFengJ9** [comment]
Verified manually that https://github.com/eclipse/openj9/pull/4919 makes this test pass.

**Comment by pshipton** [comment]
#4919 has been merged.
#5067 should be merged shortly for the equivalent fix in the 0.13 release branch
